### PR TITLE
Pdd module upgrade #154

### PR DESCRIPTION
**Pull request by leeinsunny**
oh my god I already methion all about things in previous comment

### CODE REVIEW

**Comment by leeinsunny** [comment]
In the pdd module, I added the 3 pattern dictionary

Phone number without dash
Email
the dictionary changes the email address like this
ex: [king_wang@naver.com](mailto:king_wang@naver.com) -> ******************@
the number of all characters before and after @, and put @ at last
Company Registration number
the dictionary changes the CRN like this
ex:123-45-67890 -> **********
In the dictionary, I use '?' and permit without '-' like this
ex:1234567890 -> **********
image
image
image

File: dt-demo-be/modules/pdd/run.py [hunk 1]
```diff
@@ -12,6 +12,21 @@
...
+    },
+    "phone_num_without_dash": {
+        "pattern": r'010\d{8}',
+        "type": "전화번호",
+        "replacement": "010********"
+    },
+     "email": {
+        "pattern": r'\b[\w.%+-]+@[\w.-]+\.[a-zA-Z]{2,}\b',
+        "type": "이메일 주소",
+        "replacement": lambda match: '*' * (match.end() - match.start() - 1) + '@'
```
Comment (doorooful) [inline dt-demo-be/modules/pdd/run.py:24]: Please remain email url domain link(i.e. 123@naver.com -> ***@naver.com)

File: dt-demo-be/modules/pdd/run.py [hunk 2]
```diff
@@ -12,6 +12,21 @@
...
+    },
+    "company_num": {
+        "pattern": r'\b\d{3}-?\d{2}-?\d{5}\b',
+        "type": "사업자 등록번호",
+        "replacement": "**********"
```
Comment (doorooful) [inline dt-demo-be/modules/pdd/run.py:29]: Can you follow consistency of the pattern?
(i.e. ***-**-*****)

File: dt-demo-be/modules/run.py [hunk 3]
```diff
@@ -3,19 +3,19 @@
...
-import modules.slang.run as slang
-import modules.pdd.run as pdd
-import modules.dup.run as dup
-import modules.typo.run as typo
-import modules.spc.run as spc
+#import modules.slang.run as slang
+#import modules.pdd.run as pdd
+#import modules.dup.run as dup
+#import modules.typo.run as typo
+#import modules.spc.run as spc
```
Comment (doorooful) [inline dt-demo-be/modules/run.py:10]: Please revert these changes.

**Review by doorooful — CHANGES_REQUESTED** [review]
Please refer to the comments :)

File: dt-demo-be/modules/pdd/run.py (hunk 1 above)
Comment (leeinsunny) [reply dt-demo-be/modules/pdd/run.py:24]: Okay I will make it!

File: dt-demo-be/modules/pdd/run.py (hunk 2 above)
Comment (leeinsunny) [reply dt-demo-be/modules/pdd/run.py:29]: So you mean subtract all the '-" in the middle of the company registration number from the forward and change the remaining numbers to '*'?

File: dt-demo-be/modules/run.py (hunk 3 above)
Comment (leeinsunny) [reply dt-demo-be/modules/run.py:10]: Okay! Sorry for missing this part again....

**Comment by leeinsunny** [comment]
I changed code based on dooroo's comment
image

image

image

**Comment by doorooful** [comment]
Closed as https://github.com/doorooful/CapstoneProject/pull/172 merged.
